src/study.js

```diff
@@ -8,10 +8,19 @@ import * as pageMetrics from "../src/generated/page.js";
 import * as pageVisitMetrics from "../src/generated/pageVisit.js";
 import * as eventMetrics from "../src/generated/event.js";
 import * as pageAttentionMetrics from "../src/generated/pageAttention.js";
+import * as rallyManagementMetrics from "../src/generated/rally.js";
 import * as rs01Pings from "../src/generated/pings.js";
 
+function setRallyID(rally, devMode) {
+  const rallyID = devMode ? "00000000-0000-0000-0000-000000000000" : rally._rallyID;
```

Comment:
I think this is fine for now, we should really add a public getter to `rally.js` though so it'd not necessary to get private properties like this.